openslides/core/static/templates/index.html

```diff
@@ -144,16 +144,28 @@ <h3 class="modal-title" translate>Please sign in!</h3>
           <div ui-view></div>
         </div>
       </div>
-      <hr />
-      <footer>
-        <small>
-          &copy; Copyright 2011-2015 |
-          Powered by <a href="http://openslides.org" target="_blank">OpenSlides</a> |
-          <a ui-sref="version">Version</a>
-        </small>
-      </footer>
     </div><!--/#content-->
   </div><!--/.row-->
+
+  <div ng-controller="ChatMessageCtrl">
+    <ul>
+        <li ng-repeat="chatmessage in chatmessages">
+            {{ chatmessage.user.get_full_name() }} says {{ chatmessage.message }} on {{ chatmessage.timestamp }}
+    </ul>
```

Comment:
I can't see the li elements in your template draft. Maybe `chatmessages` is still empty? Can you please check it?